tests/DCPS/QoS_XML/dumpMemBuf/run_test.pl

```diff
@@ -0,0 +1,20 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+     & eval 'exec perl -S $0 $argv:q'
+     if 0;
+
+# -*- perl -*-
+
+use lib "$ENV{ACE_ROOT}/bin";
+use PerlACE::TestTarget;
+
+my $program = PerlACE::TestTarget::create_target (1) || die "Create target 1 failed\n";
```

Comment:
- Since it's adding a new run_test.pl that file has to be listed in dcps_tests.lst in order to be included in the test suite.
- It would be good to use the new TestFramework Perl class instead of ACE's.  See `tests/DCPS/GuardCondition/run_test.pl` for an example.